vars.jsonnet

```diff
@@ -25,6 +25,11 @@
       enabled: false,
       file: import 'modules/metallb.jsonnet',
     },
+    {
+      name: 'nginxExporter',
+      enabled: true,
```

Comment:
Seeing we we don't install both Nginx & Traefik by default, should we default this to false?

Reply:
Agreed, changed it to false.